### PR TITLE
Adding supertree

### PR DESCRIPTION
**Pull request by pplonski**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pplonski** [comment]
@conda-forge/help-python, ready for review!

**Comment by pplonski** [comment]
any update? thank you!

File: recipes/supertree/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Copyright (c) MLJAR sp. z o. o.
```
Comment (ocefpaf) [inline recipes/supertree/LICENSE.txt:1]: All the license files are in the tarball. You don't need to add them here again, just list them in the recipe and conda-build will do the right thing.

Comment (pplonski) [reply recipes/supertree/LICENSE.txt:1]: Thank you. I removed license files. I don't know where to put licenses list in the recipe, something like `license_file: LICENSE.txt super tree-non-commercial-license.pdf supertree-commercial-license.pdf`?

File: recipes/supertree/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/mljar/supertree
+  summary: Interactive Decision Tree Visualization
+  license: LicenseRef-LICENSE.txt
+  license_file: LICENSE.txt
```
Comment (ocefpaf) [inline recipes/supertree/meta.yaml:39]: For the extra files you can do something like this.

```suggestion
  license_file:
    - LICENSE.txt
    - file2
    - file3
```

File: recipes/supertree/LICENSE.txt (hunk 1 above)
Comment (ocefpaf) [reply recipes/supertree/LICENSE.txt:1]: This file is still here.

Comment (pplonski) [reply recipes/supertree/LICENSE.txt]: oh, got it. thank you!

File: recipes/supertree/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - pip
+
+about:
+  home: https://github.com/mljar/supertree
+  summary: Interactive Decision Tree Visualization
+  license: LicenseRef-LICENSE.txt
+  license_file: 
+    - LICENSE.txt
+    - supertree-non-commercial-license.pdf 
+    - supertree-commercial-license.pdf
```
Comment (ocefpaf) [inline recipes/supertree/meta.yaml:42]: I'm not a lawyer and I'm always a bit scared o licenses like these. See https://github.com/mljar/supertree/blob/main/supertree-commercial-license.pdf for the full text.

@conda-forge/core any advice here? While conda-forge would not use this in a comercial sense, some users may and, in that case, who is liable? The package provider or just the end user?

Comment (pplonski) [reply recipes/supertree/meta.yaml:42]: Hi @ocefpaf and @xhochy,

Thanks for checking the package and licenses. As far as I know conda-forge is a distribution channel. It is not a part in the license. License is provided by MLJAR Sp. z o.o. to the end users. 

What is the main concern to merge the package? Thank you!

Comment (xhochy) [reply recipes/supertree/meta.yaml:42]: Distribution must be explicitly included in a license; otherwise, we are not legally allowed to do it.

Comment (pplonski) [reply recipes/supertree/meta.yaml:42]: Thank you @xhochy. We will update license and let you know.

**Comment by pplonski** [comment]
I'm not sure why windows build failed?

**Comment by pplonski** [comment]
Looks like it is working :)

**Review by xhochy — APPROVED** [review]

**Pull request merged**